src/client/java/minicraft/network/Analytics.java

```diff
@@ -52,6 +53,7 @@ public enum Analytics {
 
 	@Nullable public Future<HttpResponse<Empty>> ping() { return ping(1); }
 	@Nullable public Future<HttpResponse<Empty>> ping(int value) {
+		if (Game.debug) return null;
 		final String url = "https://pingdat.io?t="+token+"&v="+value;
 
 		return Unirest.get(url).asEmptyAsync(new Callback<Empty>() {
```

Comment:
I'm unsure how legal this class in in the EU. It's probably legal in the USA, since it has been implemented, but I am pretty sure this is a breach of the GDPR act.

Reply:
I think we should have privacy policy and Terms of Services or EULA for this.

Reply:
Doesn't matter. Still illegal.

Reply:
I am not sure about that, but that was existed for a long time.

Reply:
How should we handle this then?